templates/vm/vm.py

```diff
@@ -177,6 +178,11 @@ def enable_vnc(self):
         self._vm_sal.enable_vnc()
 
     def info(self, timeout=TIMEOUT_DEPLOY):
+        if not self.data['info']:
```

Comment:
depending on this code, I dont knw what is the expected result in the following scenario:
Make a new change in the vm like attaching a new vdisk ... How would that reflect on the `vm.info()` as the `data[info'] ` with have the old data and it will return it

Reply:
There is no action to add/remove disk on this template.
but yeah if something is changed we need to invalidate what's in data['info'] but I don't think there is

Reply:
we can do that by changing the data and call `install` action again .. right?

Reply:
you cannot change the data. that would mean you create a new service